src/metabase/types.cljc

```diff
@@ -168,9 +182,13 @@
 
 (derive :type/PostgresEnum :type/Text)
 
+(derive :type/OracleCLOB :type/Text)
+(derive :type/OracleCLOB :type/Large)
+
 ;;; DateTime Types
 
 (derive :type/Temporal :type/*)
+(derive :type/Temporal :type/field-values-unsupported)
```

Comment:
`:type/Temporal` is the only type that derives from `:type/field-values-unsupported` directly. All other types derive from `:type/Large`.